### PR TITLE
Add ping resource and test

### PR DESCRIPTION
**Pull request by nlisgo**

### CODE REVIEW

**Comment by giorgiosironi** [comment]
```
< HTTP/1.1 200 OK
* Server nginx/1.4.6 (Ubuntu) is not blacklisted
< Server: nginx/1.4.6 (Ubuntu)
< Content-Type: text/plain; charset=UTF-8
< Transfer-Encoding: chunked
< Connection: keep-alive
< Vary: Accept-Encoding
< Cache-Control: must-revalidate, no-cache, private
< X-UA-Compatible: IE=edge
< Content-language: en
< X-Content-Type-Options: nosniff
< X-Frame-Options: SAMEORIGIN
< Expires: Sun, 19 Nov 1978 05:00:00 GMT
< X-Generator: Drupal 8 (https://www.drupal.org)
< Date: Mon, 27 Mar 2017 15:52:23 GMT
< 
* Connection #0 to host ci--journal-cms.elifesciences.org left intact
pong
```
It seems that `no-store` is the only option being missing. Maybe it should be added to the non-cached responses of journal-cms in general? /cc @thewilkybarkid

**Comment by giorgiosironi** [comment]
Coveralls makes almost all builds red and gets ignored, is it that useful to report that the code coverage is at 2.359%? :thinking:

**Comment by thewilkybarkid** [comment]
Re `no-store`, don't generally make a difference as there's no chance of revalidation of such a response. We should include it here though as we really really _really_ don't want these responses ever cached anywhere.

File: src/modules/jcms_ping/src/Controller/PingController.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+namespace Drupal\jcms_ping\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Description.
+ */
+class PingController extends ControllerBase implements ContainerInjectionInterface {
```
Comment (thewilkybarkid) [inline src/modules/jcms_ping/src/Controller/PingController.php]: Are either `ControllerBase` or `ContainerInjectionInterface` really needed?

**Comment by nlisgo** [comment]
This is the response I am getting locally now: 

```
HTTP/1.1 200 OK
Server: nginx/1.10.3
Content-Type: text/plain; charset=UTF-8
Transfer-Encoding: chunked
Connection: keep-alive
Vary: Accept-Encoding
X-Powered-By: PHP/7.1.3-2+deb.sury.org~xenial+1
Cache-Control: must-revalidate, no-cache, no-store, private
X-UA-Compatible: IE=edge
Content-language: en
X-Content-Type-Options: nosniff
X-Frame-Options: SAMEORIGIN
Expires: Sun, 19 Nov 1978 05:00:00 GMT
X-Generator: Drupal 8 (https://www.drupal.org)
Date: Tue, 28 Mar 2017 13:35:42 GMT
```

**Pull request merged**